app/src/main/java/io/intrepid/contest/screens/contestcreation/NewContestActivity.java

```diff
@@ -132,6 +132,7 @@ public void showContestSubmissionPage(int page) {
     public void completeEditForm(Contest contest) {
         Toast.makeText(this, contest.getTitle() + " created ", Toast.LENGTH_SHORT).show();
         startActivity(new Intent(this, EntryImageActivity.class));
+        presenter.submitContest();
```

Comment:
Is there a reason that this is here? The view shouldn't be deciding to submit the contest; it should just tell the view that the edit form was completed and the presenter should decide whether or not to submit the contest.